src/Repository/SVNRepository.php

```diff
@@ -137,7 +137,7 @@ public function hasProviders() {
 
 	//protected function configurePackageWithVendorTransportOptions(); //@TODO come back to this
 
-	public function whatProvides( Pool $pool, $name ) {
+	public function whatProvides( Pool $pool, $name, $byPassFilters = false ) {
```

Comment:
(Although I think the variable name doesn't matter, but I'd prefer it be consistent.)